enarx-keep/src/binary/component.rs

```diff
@@ -47,7 +47,7 @@ impl Component {
 
         // Validate header assumptions.
         assert_eq!(elf.header.e_machine, EM_X86_64);
-        assert_eq!(elf.header.e_version, EV_CURRENT as _);
+        assert_eq!(elf.header.e_version, u32::from(EV_CURRENT));
```

Comment:
I'm going to be pernickety and ask for some documentation of the different constants here.  Did some quick grepping at the /enarx level, and didn't easily find a single source of truth.  I'm assuming that it comes from the goblin crate, but I think it's fair to assume that readers may not easily map these constants.